pkg/ocm/ocm_test.go

```diff
@@ -120,9 +120,9 @@ func TestClient_GetComponentVersion(t *testing.T) {
 						".dockerconfigjson": []byte(`{
   "auths": {
     "ghcr.io": {
-      "username": "skarlso",
+      "username": "open-component-model",
       "password": "password",
-      "auth": "c2thcmxzbzpwYXNzd29yZAo="
+      "auth": "b3Blbi1jb21wb25lbnQtbW9kZWw6cGFzc3dvcmQ="
```

Comment:
@Skarlso - is this fake/test data? Or real working credentials? If the second is the case, can we come up with a solution without publishing them?

@morrison-sap f.y.i.

Reply:
It's base64 encoded test data.